src/main/java/org/opentripplanner/routing/graph/Graph.java

```diff
@@ -320,6 +354,8 @@
      */
     public RouterHealth routerHealth = new RouterHealth();
 
+    public StreetHeuristicData streetHeuristicData;
```

Comment:
?

Reply:
it is set in graph builder module and used during heuristic initialization